aksel.nav.no/website/scripts/changelog.ts

```diff
@@ -40,21 +41,66 @@ export async function main() {
 
   const blockContentType = defaultSchema
     .get("blogPost")
-    .fields.find((field) => field.name === "body").type;
-
-  const blocks = htmlToBlocks(html, blockContentType, {
-    parseHtml: (htmlStr) => new JSDOM(htmlStr).window._document,
-  });
-  blocks.shift();
+    .fields.find((field: any) => field.name === "body").type;
 
-  client
-    .patch("75351662-ecb0-455a-b9a5-59f319e423fa")
-    .set({ content: blocks })
-    .commit()
-    .then(() => {
-      console.info("Hurray, updated changelog!");
-    })
-    .catch((err) => {
-      console.error("Oh no, the changelog-update failed: ", err.message);
+  // We go through the last 3 entries in case there has been changes,
+  // or in case previous attempts have failed.
+  for (let i = 0; i < 3; i++) {
```

Comment:
If im understanding this correctly, the "oldest" changelog entry might be created as the "newest" now. Consider reversing the loop to start witht he oldest 
```
  for (let i = 3; i >= 0; i--) {
```